packages/frontend-2/components/server-management/Table.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="text-foreground">
+    <div class="w-full text-sm overflow-x-auto overflow-y-visible simple-scrollbar">
+      <div
+        class="grid z-10 grid-cols-12 items-center gap-6 font-semibold bg-foundation rounded-t-lg w-full border-b border-outline-3 pb-2 pt-4 px-4 min-w-[900px]"
+        :style="{ paddingRight: paddingRightStyle }"
+      >
+        <div
+          v-for="header in headers"
+          :key="header.id"
+          :class="columnClasses[header.id]"
+          class="capitalize"
+        >
+          {{ header.title }}
+        </div>
+      </div>
+      <div
+        class="divide-y divide-outline-3 h-full overflow-visible"
+        :class="{ 'pb-32': overflowCells }"
+      >
+        <div
+          v-for="item in items"
+          :key="item.id"
+          class="relative grid grid-cols-12 items-center gap-6 px-4 py-1 min-w-[900px] bg-white"
```

Comment:
There still seem to be various non-semantic styles that look broken in dark mode, please validate everything looks OK in both themes
<img width="994" alt="image" src="https://github.com/specklesystems/speckle-server/assets/938316/ee930e0a-dde2-4460-9d9e-ba2e25850a05">